### DISCUSSION TITLE
Bazel support for pkl

### DISCUSSION THREAD

**Discussion by dieu** [apple/pkl#256] [Ideas]
Hello,

Is there any appetite for creating Bazel rules for all?

**Comment by bioball** [comment]
Yes, this is something that we are working on. I can't provide an estimate yet for when we will release it, though.

**Reply by dieu** [reply]
Thanks for update, is anything to do for community to help with implementation?

**Reply by storm-of-fists** [reply]
@dieu you wanna help me figure out why my rules here can't detect the toolchains im registering? https://stackoverflow.com/q/78113876/14227639

**Comment by storm-of-fists** [comment]
I have an appetite, working on it this week hopefully. https://github.com/storm-of-fists/rules_pkl

**Reply by tusharkumarsahoo** [reply]
Not able to access the repo 🤔

![image](https://github.com/apple/pkl/assets/66913648/49da9309-660d-4a54-805f-4f60a1b3adde)

**Comment by holzensp** [comment]
We have now [released initial Bazel integration, through `rules_pkl`](https://github.com/apple/rules_pkl/). Contributions / improvements more than welcome!
